### PR TITLE
Avoid creating the nilearn_cache folder when using nilearn maskers

### PR DESCRIPTION
**Pull request by vinferrer**
Changes proposed in this pull request:

- Now the memory option in the masker is not used
-

### CODE REVIEW

File: connPFM/connectivity/ev.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def event_detection(data_file, atlas, surrprefix="", sursufix="", nsur=100, segm
     masker = NiftiLabelsMasker(
         labels_img=atlas,
         standardize=False,
-        memory="nilearn_cache",
+        memory="",
```
Comment (eurunuela) [inline connPFM/connectivity/ev.py:57]:
```suggestion
```

File: connPFM/debiasing/debiasing.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def debiasing(data_file, mask, mtx, tr, out_dir, history_str):
     masker = NiftiLabelsMasker(
         labels_img=mask,
         standardize=False,
-        memory="nilearn_cache",
+        memory="",
```
Comment (eurunuela) [inline connPFM/debiasing/debiasing.py:22]:
```suggestion
```

File: connPFM/connectivity/ev.py (hunk 1 above)
Comment (eurunuela) [reply connPFM/connectivity/ev.py:57]: The default is no cache.

File: connPFM/deconvolution/roiPFM.py [hunk 3]
```diff
@@ -38,9 +38,7 @@ def roiPFM(
     LGR.info("Masking data...")
     atlas_old = atlas
     atlas = atlas_mod.transform(atlas, data, dir)
-    masker = NiftiLabelsMasker(
-        labels_img=atlas, standardize="psc", memory="nilearn_cache", strategy="mean"
-    )
+    masker = NiftiLabelsMasker(labels_img=atlas, standardize="psc", memory="", strategy="mean")
```
Comment (eurunuela) [inline connPFM/deconvolution/roiPFM.py:41]:
```suggestion
    masker = NiftiLabelsMasker(labels_img=atlas, standardize="psc", strategy="mean")
```

File: connPFM/tests/test_ev.py [hunk 4]
```diff
@@ -10,7 +10,7 @@ def test_calculate_ets(ets_auc_original_file, AUC_file, atlas_file):
     masker = NiftiLabelsMasker(
         labels_img=atlas_file,
         standardize=False,
-        memory="nilearn_cache",
+        memory="",
```
Comment (eurunuela) [inline connPFM/tests/test_ev.py:13]:
```suggestion
```

File: connPFM/tests/test_ev.py [hunk 5]
```diff
@@ -25,7 +25,7 @@ def test_rss_surr(AUC_file, atlas_file, surr_dir, rssr_auc_file):
...
```
Comment (eurunuela) [inline connPFM/tests/test_ev.py:28]:
```suggestion
```

File: connPFM/utils/surrogate_generator.py [hunk 6]
```diff
@@ -36,7 +36,7 @@ def generate_surrogate(data, atlas, output):
     # Mask data
     LGR.info("Masking data...")
     surrogate_masker = NiftiLabelsMasker(
-        labels_img=atlas, standardize="psc", memory="nilearn_cache", strategy="mean"
+        labels_img=atlas, standardize="psc", memory="", strategy="mean"
```
Comment (eurunuela) [inline connPFM/utils/surrogate_generator.py:39]:
```suggestion
        labels_img=atlas, standardize="psc", strategy="mean"
```

File: ismrm2022/stability_pfm.py [hunk 7]
```diff
@@ -35,7 +35,7 @@ def main():
     masker = NiftiLabelsMasker(
         labels_img=atlas,
         standardize=False,
-        memory="nilearn_cache",
+        memory="",
```
Comment (eurunuela) [inline ismrm2022/stability_pfm.py:38]:
```suggestion
```

**Review by eurunuela — CHANGES_REQUESTED** [review]
LGTM @vinferrer . I would remove the memory argument though, as the default is cache. I think it would make it easier to read.

**Comment by vinferrer** [comment]
is done

**Review by eurunuela — APPROVED** [review]
LGTM, thank you @vinferrer !

**Pull request merged**